.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+cache: pip
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
```

Comment:
Drop 3.4 because it is EOL and Add 3.7?

Reply:
Thanks. 3.7 is now running on Travis.

I'm not sure if it makes sense to include multi-browser testing as part of this matrix, or just test multiple browsers within each python-version matrix job...